Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -83,6 +87,8 @@ public static IServiceCollection AddWebServer<TStartup>(this IServiceCollection
                 .ConfigureApplicationCookie(opt =>
                 {
                     opt.Cookie.Domain = EnvironmentUtils.GetRequiredByKey(SharedAuthCookieDomain);
+                    opt.Events.OnRedirectToAccessDenied = UnAuthorizedResponse;
+                    opt.Events.OnRedirectToLogin = UnAuthorizedResponse;
```

Comment:
Why do we return unauthorized when redirect to login?

Reply:
Because the [Authorize[ attribute redirects to /Login and login returns the Index view and this way we can never get to a 401 Response from our API and take actions on the FE client based on the http response because its always 200OK with the index html

Reply:
Will allowing our API for all users and removing the unauthorized logic fix this problem?

Reply:
Its good to have this as this will occur in other cases. No point in having a GET Login endpoint in the backend and redirecting to it, because this way the FE client cannot determine unauthorized scenario.